.ci/packer_cache.sh

```diff
@@ -15,5 +15,30 @@ while [ -h "$SCRIPT" ] ; do
   fi
 done
 
-source $(dirname "${SCRIPT}")/java-versions.properties
-JAVA_HOME="${HOME}"/.java/${ES_BUILD_JAVA} ./gradlew resolveAllDependencies --parallel
+cd $(dirname "${SCRIPT}")/..
+
+resolveAllDependencies() {
+    local REFRESH_FAILED=no
+    local MAX_TRIES=3
+    for i in $(seq 1 $MAX_TRIES) ; do 
```

Comment:
I don't think we need to retry. That can be done externally, building the packer image should fail if this script fails, and then rebuilding retried.